src/libraries/Directory.Build.props

```diff
@@ -348,9 +271,6 @@
     <!-- setting the output paths -->
     <OutputPath>$(BaseOutputPath)$(TargetFramework)-$(TargetFrameworkSuffix)-$(Configuration)\</OutputPath>
     <OutputPath Condition="'$(TargetFrameworkSuffix)' == ''">$(BaseOutputPath)$(TargetFramework)-$(Configuration)\</OutputPath>
-    <BaseIntermediateOutputPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', '$(MSBuildProjectName)'))</BaseIntermediateOutputPath>
```

Comment:
This should remain I think since it doesn’t need TargetFramework.

Reply:
The reason for move is same as https://github.com/dotnet/runtime/pull/34025#discussion_r400626301

It will be nice to keep all IntermediateOutput path properties together.